tools/lkl/lib/hijack/hijack.c

```diff
@@ -338,3 +341,14 @@ int eventfd_write(int fd, uint64_t value)
 	return lkl_sys_write(fd, (void *) &value,
 			     sizeof(value)) != sizeof(value) ? -1 : 0;
 }
+
+HOST_CALL(mmap)
+void *mmap(void *addr, size_t length, int prot, int flags, int fd, off_t offset)
+{
+	CHECK_HOST_CALL(mmap);
+
+	if (addr != NULL || flags != (MAP_ANONYMOUS|MAP_PRIVATE) ||
```

Comment:
The libc we are using calls mmap in malloc which get hijacked with this change. Maybe you want to use fd != -2